src/EnergyPlus/ThermalComfort.cc

```diff
@@ -2308,6 +2309,10 @@ namespace ThermalComfort {
 				} else {
 					deltaT = ZTAV( iZone ) - ZoneThermostatSetPointHi( iZone );
 				}
+				
+				if ( Zone( iZone ).HasApproachTempToReturnAir ) {
+					deltaT = TempTstatAir( iZone ) - Zone( iZone ) .AdjustedTempToReturnAir;
+				}
```

Comment:
This implies that the thermostat is now controlling on return air temperature.  That doesn't seem correct.

Reply:
As we discussed before. The room temperature will converge to the return air temperature for heat balance.